openapi/index.yaml

```diff
@@ -1052,6 +1052,296 @@ paths:
           description: Internal server error
           schema:
             $ref: '#/definitions/ProblemJson'
+  /service-messages/manage/api/v1/remote-contents/configurations:
```

Comment:
We could have /api/v1/manage/messages/ as the path prefix for any message management API?
Could the final path for the remote content configuration API be something like this /api/v1/manage/messages/remote-content/configurations?
My point is:
- be as conservative as possible with respect to other actual APIs
- have a self-explanatory path 
Perhaps @rodoviti can help us as he has already addressed this with the services manage api

And in a future step, we'll decide how to manage different stream APIs.

Reply:
maybe it's easier to have something like /api/v1/communication/manage/remote-content/configurations

Reply:
"manage" is superfluous imho since the action is implied by the http verb which acts on the configuration resource.

Reply:
Actually I agree, we can't use /api/v1/messages as a path prefix because it matches other existing endpoints with different backends, we could use a messages synonym (e.g. /api/v1/communication/remote-content/configurations) or something else and 'manage' is similar to others (a general revision is needed to enable a better api stream ownership)

Reply:
This part of the path ` /service-messages/manage` is made by 2 reasons:
1. we migrated the `service-message` part from AKS to Azure Functions and to have a 0 downtime we maintained the original openapi specs.
2. we added `manage` because we are exposing these specs to TPs and we had to distinguish from the internal use of these APIs at the APIM's level.

We could now achieve what you are saying if it is really necessary, we should delay the delivery to TPs of these APIs that was for the next week because we have now to prepare a new openapi without the ` /service-messages/manage` part for both the APIs, and then create 2 new APIs on APIM distinguishing the internal ones with a `internal` subpath, and then deploy to the Azure Functions the new functions with new paths without the `service-messages` part.